x-pack/plugin/ml/qa/single-node-tests/build.gradle

```diff
@@ -1,13 +1,6 @@
-apply plugin: 'elasticsearch.testclusters'
-apply plugin: 'elasticsearch.standalone-rest-test'
-apply plugin: 'elasticsearch.rest-test'
+apply plugin: 'elasticsearch.java-rest-test'
 
-dependencies {
-  testImplementation project(":x-pack:plugin:core")
-  testImplementation project(path: xpackModule('ml'))
```

Comment:
here is an example where duplicating the BASE_PATH allows us to remove the dependency.